### PR TITLE
Add strip_workspace_name

### PR DESCRIPTION
**Pull request by hwangcc23**
See the issue #3163 (https://github.com/i3/i3/issues/3163).

Add strip_workspace_name to strip off the workspace name.

### CODE REVIEW

**Comment by Airblader** [comment]
Thanks for the PR! I'll wait with merging unlike the next release we are currently planning (merge freeze).

File: i3bar/src/workspaces.c [hunk 1]
```diff
@@ -106,8 +106,8 @@ static int workspaces_string_cb(void *params_, const unsigned char *val, size_t
         const char *ws_name = (const char *)val;
         params->workspaces_walk->canonical_name = sstrndup(ws_name, len);
 
-        if (config.strip_ws_numbers && params->workspaces_walk->num >= 0) {
-            /* Special case: strip off the workspace number */
+        if ((config.strip_ws_numbers | config.strip_ws_name) && params->workspaces_walk->num >= 0) {
```
Comment (Airblader) [inline i3bar/src/workspaces.c]: I think a bitwise OR is a bit weird here. Can we just use the normal logical or?

Comment (hwangcc23) [reply i3bar/src/workspaces.c]: Sorry. My fault. It is a typo.
Fixed.

**Comment by Airblader** [comment]
Thanks!

**Comment by ckdarby** [comment]
@hwangcc23 Thanks, I appreciate you resolving this for me. I looked at trying to contribute these changes myself but couldn't get it to make properly. I also don't do much C development.

**Pull request merged**